docs/authentication/session-management-overview.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Session Management overview
+id: session-management-overview
+description: ''
+slug: /session-management-overview
+keywords:
+  - api token
+  - access token
+  - session management
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 08/08/2023
+  author: William May
+doc_type: overview
+displayed_sidebar: mainSidebar
+---
+
+import MultiLanguageCodeBlock from '@site/src/components/CodeBlocks/MultiLanguageCodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What is Session Management
+
+Session management is a crucial aspect of web application development. An authentication session is a period of user interaction with the application, typically starting with the user providing credentials and ending with logout or inactivity timeout. Effective session management ensures user data privacy, maintains session state, and provides a seamless user experience.
+
+The Beyond Identity session consists of something we call a `token`, specifically [access token](/docs/next/api-token-overview)s. An access token, also called an API token, is an object or file that enables access to an application or resource. The token is a JWT, which is digitally signed and contains information about what resources can be accessed and for how long.
+
+When you request an [access token](https://datatracker.ietf.org/doc/html/rfc6749#section-1.4), it is possible to receive an additional token called a [refresh token](https://datatracker.ietf.org/doc/html/rfc6749#section-1.5) in the response. The refresh token can be used to request additional access tokens with identical or narrower scope.
+
+## How to add session management to my application
+
+There are a few functions of tokens that you should be aware of.
+
+- [Create a Token](/docs/next/create-api-token) (Pre-Requisite)
+- [Store Tokens](#how-to-store-tokens)
+- [Introspect a Token](/docs/next/validate-access-tokens)
+- [Revoke a Token](/docs/next/revoke-access-tokens)
+- [Retrieve Active Tokens](/docs/next/list-access-tokens)
+
+## How to store tokens
+
+Applications may utilize cookies, local storage or other strategies to associate session with the end-user’s application. Beyond Identity doesn’t provide an API/SDK for this purpose; it is up to you to decide how to handle the end-user’s session. One option is to store the JWT received from Beyond Identity in the end user’s application. Another solution is to store the JWT received from Beyond Identity on the server and use a cookie. This is a common solution offered by many web frameworks and is more secure as the JWT is more sensitive.
+
+### JWT Strategy with Refresh/Access Tokens
+
+In the context of session management, the central element is the JSON Web Token (JWT), which serves as the token that encapsulates and conveys the authentication and authorization information necessary for maintaining secure user sessions.
```

Comment:
Do you know what claims are stored in the JWT for session tokens?

Reply:
I couldn't find good documentation on the claims. There are two examples (one [with](https://developer-docs-git-sessionmanagement-beyondidentity.vercel.app/docs/next/create-api-token#example-create-a-token-with-custom-claims) custom claims and one [without](https://developer-docs-git-sessionmanagement-beyondidentity.vercel.app/docs/next/create-api-token#example-create-a-token-with-a-limited-expiration-time)) in Create an access token, but I don't know what each claim means

Reply:
maybe @tomsik68 knows which claims we currently support. Might be nice to list those